internal/fs/fs.go

```diff
@@ -986,9 +986,8 @@ func (fs *fileSystem) syncFile(
 	ctx context.Context,
 	f *inode.FileInode) (err error) {
 	// syncFile can be triggered for unlinked files if the fileHandle is open by
-	// another user/terminal. Hence, ignoring the syncFile call for local file.
```

Comment:
filehandle can be open in same terminal as well. Just mention same user or another user

Reply:
done